test/src/com/xilinx/rapidwright/router/TestSATRouter/reduce_or_routed_7overlaps_solution.txt

```diff
@@ -0,0 +1,3491 @@
+{
```

Comment:
Any particular reason to keep this in a the `src` folder?  It seems like `test/RapidWrightDCP` would be more appropriate and has other `.txt` files in that location as well.